crates/core/src/metadata/manager.rs

```diff
@@ -54,7 +54,7 @@ enum UpdateError {
     Network(#[from] NetworkError),
 }
 
-#[derive(Debug)]
+#[derive(Copy, Clone, Debug, PartialEq, Eq, Ord, PartialOrd, derive_more::Display)]
```

Comment:
Why does this need Ord/PartialOrd?

Reply:
I mindlessly copied it over. Will remove it.